client/app/components/form/UserOperatorMultiStepForm.tsx

```diff
@@ -78,17 +80,26 @@ export default function UserOperatorMultiStepForm({
     );
   };
 
+  const isAdmin = session?.user.app_role?.includes("cas");
```

Comment:
would recommend renaming this variable to avoid confusion - both `cas_admin` and `cas_analyst` roles will be reviewing prime admin requests, and we'll also have a role called `industry_user_admin` that should not be allowed to review themselves. Rename to something like `isCasInternal` maybe?

the other thing to note is the default app_role when authenticating with IDIRs is `cas_pending`, who should not be allowed to do reviews until they've had their role elevated to either analyst or admin, so we'll need to change the logic on how the app_role is checked

Reply:
I have renamed it to isCasInternal and adjusted the logic but I believe a cas-pending user is already blocked form reviewing from the page authentication not allowing them to get there in the first place.